Users/Glob_Patterns.rst

```diff
@@ -174,14 +209,10 @@ brackets. Parentheses that have no match are ignored as well as
 
```

Comment:
But following the wildcards of os , we don't need to use \ before each *.
**/ should be fine.